src/makeCalculator.js

```diff
@@ -38,6 +38,34 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    add(num) {
+      this.result += num;
+      return this.result;
+    },
+    subtract(num) {
```

Comment:
add spaces between blocks
```suggestion
    },

    subtract(num) {
```